src/app/game/schulte-table/page.tsx

```
@@ -0,0 +1,156 @@
+        </ul>
+
+        <button
+          className={styles.learnBtn}
+          onClick={() =>
+            window.open(
+              'https://www.geeksforgeeks.org/dsa/shuffle-a-given-array-using-fisher-yates-shuffle-algorithm/',
+              "_blank", "noopener,noreferrer" 
+            )
+          }
+        >
+          Learn more about it
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)